ENMeval 2.0.0 update

jamiemkass opened the pull request:
I made some changes to `BIOMOD_tuning()` to accommodate the new `ENMeval` 2.0.0. I removed the custom implementation and replaced it with a single call to `ENMevaluate()`, which I think does the same thing, but please check to make sure it does, as I do not know what the intentions of the custom implmentation were. I also updated the column names referenced internally, which seem to also be out of date for `ENMeval` 0.3.1. Please let me know if you have any questions.

Jamie

MayaGueguen commented:
Thank you Jamie ! Two comments :
-  man/BIOMOD_presenceonly.Rd file has been deleted, thus raising a WARNING in check
- you committed also example files (GuloGulo) and maybe it is better to keep those at out the github folder

Could you make the changes before merging ?

jamiemkass commented:
Thanks Maya, glad to be of service. I added back the .Rd file, which may have went missing when I ran `devtools::document()` because it appears there are no roxygen comments for `BIOMOD_presenceonly.R`, although the .Rd says it was written with roxygen2 (maybe from a previous version?). 

Regarding the GuloGulo data, I removed it all in a previous commit.

Best,

Jamie
